src/main/java/org/mskcc/oncokb/curation/config/security/SecurityConfigurationOAuth.java

```diff
@@ -130,12 +126,54 @@ public GrantedAuthoritiesMapper userAuthoritiesMapper() {
             authorities.forEach(authority -> {
                 // Check for OidcUserAuthority because Spring Security 5.2 returns
                 // each scope as a GrantedAuthority, which we don't care about.
-                if (authority instanceof OidcUserAuthority) {
-                    OidcUserAuthority oidcUserAuthority = (OidcUserAuthority) authority;
+                if (authority instanceof OidcUserAuthority oidcUserAuthority) {
```

Comment:
Cool - Since java 14, we can now check and cast in one line

Reply:
I think it's finalized in 16. We are aiming for 17 for this pr.